app/Http/Controllers/GuestController.php

```diff
@@ -33,13 +33,16 @@ public function rsvp(Request $request)
 	    	$guest->attending = $request->attending;
 	    	$guest->email = $request->email;
 	    	$guest->save();
+
+	    	//If guest is attending, they can invite another, else they can't.
 	    	return response()->json([ 
-	    		'success' => 'Awesome pawesome',
+	    		'success' => 'Your have been updated on the wedding list.',
+	    		'inviteAnother' => $guest->attending,
 	    		'invitorFN' => $firstName,
 	    		'invitorLN' => $lastName ]);
 		}
 
-		return response()->json([ 'error' => 'You are not registered in the database']);
+		return response()->json([ 'error' => 'Sorry, you are not registered in the database. If you received an invitation, please contact us ASAP.']);
```

Comment:
Maybe something along the lines of 
"Sorry, we could not register your process your registration. If you received an invitation, please contact us to ASAP." ?